utilix/batchq.py

```diff
@@ -56,6 +64,15 @@
     "/dali/lgrandi/grid_proxy/xenon_service_proxy:/project2/lgrandi/grid_proxy/xenon_service_proxy",
 ]
 
+class QOSNotFoundError(Exception):
```

Comment:
I think then we do not need these designed exceptions?

Reply:
I think it's still good to keep those types for future use